packages/expo-modules-core/ios/Core/Views/SwiftUI/SwiftUIHostingView.swift

```diff
@@ -44,7 +44,9 @@ extension ExpoSwiftUI {
       super.init(appContext: appContext)
 
       shadowNodeProxy.setViewSize = { size in
+        #if RCT_NEW_ARCH_ENABLED
```

Comment:
just wondering why this is? This means something will be broken on old arch, no?

Reply:
Actually it won't be broken, as this type of view will never be created on old arch, see SwiftUIViewDefinition.swift. This change just makes sure that the classes still compile on old arch.

Reply:
I'll defer to @aleqsio or @lukmccall on whether the check for new arch should be in a different place...

Reply:
I'm good with autosizingstack working on new arch only for now – it would be fairly hard to make it work on older arch and I'm not sure it's worth it – making it noop as in this PR seems fine.